src/esp/gfx/GenericDrawable.cpp

```diff
@@ -143,7 +154,7 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
               : node_.getSemanticId())
       .setTransformationMatrix(transformationMatrix)
       .setProjectionMatrix(camera.projectionMatrix())
-      .setNormalMatrix(transformationMatrix.normalMatrix());
+      .setNormalMatrix(invNormalMat.transposed());
```

Comment:
You're transposing twice, I don't think that is necessary. Just don't transpose at all.

Same for the PBR drawable.